### PR TITLE
Feature/use mep

### PR DESCRIPTION
**Pull request by mattwuenschel**
Mapr is using the mep repos for Mapr 5.2 and on instead of the ecosystem repo. 
http://package.mapr.com/releases/

### CODE REVIEW

File: recipes/repo.rb [hunk 1]
```diff
@@ -48,6 +54,17 @@
...
     action :add
+    only_if { node['hadoop_mapr']['distribution_version'] < '5.2' }
+  end
+
+  yum_repository 'mep' do
+    name 'maprmep'
+    description 'MapR Technologies'
+    url yum_ecosystem_repo_url
+    gpgkey yum_repo_key_url
+    gpgcheck false
```
Comment (wolf31o2) [inline recipes/repo.rb:65]: Are these packages not signed?

File: recipes/repo.rb [hunk 2]
```diff
@@ -30,9 +31,14 @@
   include_recipe 'yum'
 
   yum_repo_url = "http://package.mapr.com/releases/v#{mapr_release}/redhat"
-  yum_ecosystem_repo_url = "http://package.mapr.com/releases/ecosystem-#{mapr_release.to_i}.x/redhat"
   yum_repo_key_url = 'http://package.mapr.com/releases/pub/maprgpg.key'
 
+  yum_ecosystem_repo_url = if node['hadoop_mapr']['distribution_version'] >= '5.2'
```
Comment (wolf31o2) [inline recipes/repo.rb]: Shouldn't we need to make these something comparable, like floats? I think that you need to `.to_f` to convert to a float to do a >= comparison.

**Review by wolf31o2 — CHANGES_REQUESTED** [review]
Couple questions

File: recipes/repo.rb (hunk 1 above)
Comment (mattwuenschel) [reply recipes/repo.rb:65]: Mapr has the gpgcheck=0 for the MEP repos in the install instructions.
http://maprdocs.mapr.com/home/AdvancedInstallation/AddingMapRreposonRHorCOS.html

File: recipes/repo.rb [hunk 3]
```diff
@@ -48,6 +54,17 @@
     gpgkey yum_repo_key_url
     gpgcheck false
     action :add
+    only_if { node['hadoop_mapr']['distribution_version'] < '5.2' }
```
Comment (wolf31o2) [inline recipes/repo.rb]: Use float here, too.

File: recipes/repo.rb [hunk 4]
```diff
@@ -30,9 +31,14 @@
   include_recipe 'yum'
 
   yum_repo_url = "http://package.mapr.com/releases/v#{mapr_release}/redhat"
-  yum_ecosystem_repo_url = "http://package.mapr.com/releases/ecosystem-#{mapr_release.to_i}.x/redhat"
   yum_repo_key_url = 'http://package.mapr.com/releases/pub/maprgpg.key'
 
+  yum_ecosystem_repo_url = if node['hadoop_mapr']['distribution_version'].to_f >= 5.2
```
Comment (wolf31o2) [inline recipes/repo.rb]: I'd use `mapr_release.to_f` in case the version includes a 'v' as it won't convert.

File: recipes/repo.rb (hunk 3 above)
Comment (wolf31o2) [reply recipes/repo.rb]: and `mapr_release.to_f`

File: recipes/repo.rb (hunk 1 above)
Comment (wolf31o2) [reply recipes/repo.rb:65]: :+1:

File: recipes/repo.rb [hunk 5]
```diff
@@ -48,6 +54,17 @@
...
+  end
+
+  yum_repository 'mep' do
+    name 'maprmep'
+    description 'MapR Technologies'
+    url yum_ecosystem_repo_url
+    gpgkey yum_repo_key_url
+    gpgcheck false
+    action :add
+    only_if { node['hadoop_mapr']['distribution_version'] >= '5.2' }
```
Comment (wolf31o2) [inline recipes/repo.rb]: Here, too.

File: recipes/repo.rb [hunk 6]
```diff
@@ -57,9 +74,14 @@
   include_recipe 'apt'
 
   apt_repo_url = "http://package.mapr.com/releases/v#{mapr_release}/ubuntu"
-  apt_ecosystem_repo_url = "http://package.mapr.com/releases/ecosystem-#{mapr_release.to_i}.x/ubuntu"
   apt_repo_key_url = 'http://package.mapr.com/releases/pub/maprgpg.key'
 
+  apt_ecosystem_repo_url = if node['hadoop_mapr']['distribution_version'] >= '5.2'
```
Comment (wolf31o2) [inline recipes/repo.rb]: Here, too.

File: recipes/repo.rb [hunk 7]
```diff
@@ -74,5 +96,15 @@
     trusted true
     distribution 'binary/'
     action :add
+    only_if { node['hadoop_mapr']['distribution_version'] < '5.2' }
```
Comment (wolf31o2) [inline recipes/repo.rb]: Here

File: recipes/repo.rb [hunk 8]
```diff
@@ -74,5 +96,15 @@
...
+  end
+
+  apt_repository 'maprmep' do
+    uri apt_ecosystem_repo_url
+    key apt_repo_key_url
+    trusted true
+    distribution 'binary/'
+    action :add
+    only_if { node['hadoop_mapr']['distribution_version'] >= '5.2' }
```
Comment (wolf31o2) [inline recipes/repo.rb]: Here

**Review by wolf31o2 — CHANGES_REQUESTED** [review]
Few more conversions to floats needed.

**Review by wolf31o2 — APPROVED** [review]
👍 LGTM

**Pull request merged**